oksp/settings/base_settings.py

```diff
@@ -95,6 +96,8 @@
 
 USE_TZ = True
 
+MEDIA_URL = '/media/'
```

Comment:
Dont make a new variable, we have `MEDIA_ROOT` declared right below which is the same.